.github/workflows/push.yml

```diff
@@ -0,0 +1,55 @@
+name: Build and Push
```

Comment:
```suggestion
name: Build and Push to Harbor
```

Reply:
Should we really use the registry application name?

Reply:
I don't care. Ignore my suggestion.